examples/a4a-fullwidth-rtl.amp.html

```diff
@@ -0,0 +1,240 @@
+<!doctype html>
+<html ⚡ dir="rtl">
+<head>
+  <meta charset="utf-8">
+  <title>AdSense full-width ad examples</title>
+  <link rel="canonical" href="http://nonblocking.io/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-custom>
+    body {
+      background-color: #DDDDDD;
+    }
+
+    /** Put all content in unusual margins for testing. */
+    .content {
+      margin-left: 15px;
+      margin-right: 12px;
+    }
+
+    h1 {
+      padding: 12px 0;
+      margin: 0;
+    }
+
+    p {
+      margin: 0;
+      padding: 6px 2px;
+    }
+
+    h1, .everything {
+      background-color: #999999;
+    }
+
+    amp-ad div[placeholder] {
+      background-color: lightgray;
+    }
+
+    amp-ad div[fallback] {
+      background-color: black;
+    }
+
+    amp-ad div[placeholder]::after {
+      content: "loading ...";
+    }
+
+    amp-ad div[fallback]::after {
+      content: "No ad";
+      color: white;
+    }
+
+    amp-user-notification {
+      background-color: lightgray;
+      padding: 5px;
+    }
+
+    .header-img {
+      margin: 0 auto;
+      text-align: center;
+    }
+  </style>
+  <style amp-boilerplate>body {
```

Comment:
Please re-copy/paste boilerplate. We'd like it to be a non-formatted blob in our examples. Because, well, it is :)